src/templates/person.html

```diff
@@ -193,6 +193,25 @@ <h2>{{ person.email }}</h2>
       <i>This person does not have an account.</i>
     {% endif %}
 
+    <!-- Seizures Protection -->
+    <div class="panel panel-default">
+      <div class="panel-heading"><h4>Seizure Protection</h4></div>
+      <div class="panel-body">
+          <form method="POST" action="/__BACKOFFICE__/setAccountIsPkonto/{{ person.id }}">
+            <div class="form-group">
+              <label for="isPkonto">Is account P-Konto?</label>
```

Comment:
We are adding a mock sesiure data here, isPkonto can be confusing becuse real solaris doesn't support "isPkonto"